mindmeld/models/tagger_models.py

```diff
@@ -16,7 +16,7 @@
 import os
 import random
 
-from sklearn.externals import joblib
+import joblib
```

Comment:
does this work with lower version of sklearn?

maybe we can do this:
```
try: 
    import joblib
except:
    from sklearn.externals import joblib

Reply:
I'm pretty sure it does. I think the joblib package works same as it's scikit-lean equivalent. And did you mean for the try catch be the other way around? :)